Update template translation

comfyui-wiki opened the pull request:
Update all the template translations.

┆Issue is synchronized with this [Notion page](https://www.notion.so/PR-4396-Update-template-translation-22b6d73d36508132bf03cb5f76013131) by [Unito](https://www.unito.io)

comfyui-wiki committed:
Update template translation

Commit:
Update locales [skip ci]

christian-byrne commented on src/locales/es/main.json:
```diff
@@ -1239,42 +1249,64 @@
+        "image_lotus_depth_v1_1": "Lotus Depth",
+        "image_omnigen2_image_edit": "OmniGen2 Edición de Imagen",
+        "image_omnigen2_t2i": "OmniGen2 Texto a Imagen",
+        "sd3.5_large_blur": "SD3.5 Grande Desenfoque",
+        "sd3.5_large_canny_controlnet_example": "SD3.5 Grande Canny ControlNet",
+        "sd3.5_large_depth": "SD3.5 Grande Profundidad",
+        "sd3.5_simple_example": "SD3.5 Simple",
```
we are replacing dots with underscores.

comfyui-wiki committed:
Fix translation display issue.

comfyui-wiki committed:
Resolve merge conflicts by keeping original translation structure

- Accept HEAD version (720e0a99) for all locale files
- Restore Comfy_Notification_ShowVersionUpdates setting in settings.json
- Ignore GitHub Action's alphabetical reorganization
- Keep original template structure intact

comfyui-wiki committed:
Remove duplicate key

Commit:
Update locales [skip ci]

comfyui-wiki committed:
Merge branch 'main' into update-the-templates-translation

Commit:
Update locales [skip ci]

comfyui-wiki commented:
Just about to add the zh-TW translation of templates and found that the GitHub Action had already finished it. 👍

christian-byrne reviewed (commented):
So far I have been using "LoRA" instead of "Lora". I think "LoRA" is the standard ML terminology (Low Rank Adaption ...). WDYT?

comfyui-wiki committed:
Merge branch 'main' into update-the-templates-translation

comfyui-wiki committed:
Correct "lora" to "LoRA"

Commit:
Update locales [skip ci]

comfyui-wiki committed:
[i18n] Update template translations - standardize "Lora" to "LoRA" across locales

comfyui-wiki committed:
Merge branch 'update-the-templates-translation' of https://github.com/Comfy-Org/ComfyUI_frontend into update-the-templates-translation

Commit:
Update test expectations [skip ci]

christian-byrne reviewed (approved):
LGTM. Thank you!
